VideoLocker/res/layout/activity_course_base.xml

```diff
@@ -196,7 +193,9 @@
                         android:layout_height="@dimen/course_unit_nav_bar_btn_height"
                         android:background="@color/white"
                         android:text="@string/assessment_previous"
-                        android:textColor="?attr/buttonTextColor" />
+                        android:textColor="?attr/buttonTextColor"
+                        android:gravity="center_vertical|start"
```

Comment:
Not a good idea, because, we are supporting accessibility. A button should be a button class's object, so that accessibility pronounces it e.g. "Next Button" otherwise TalkBack would just say "Next", which isn't what we want.